packages/now-static-build/test/fixtures/10b-gatsby-plugin-resolution/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "private": true,
+  "scripts": {
+    "build": "gatsby build"
+  },
+  "dependencies": {
+    "gatsby": "2.17.11",
```

Comment:
Let's rename this test fixture from `10b-gatsby-plugin-resolution` to `10-gatsby217-plugin-resolution`

Reply:
I've renamed it to `10b-gatsby-21711` (kept the "b" to keep the order).